### PR TITLE
Handle new drives found by additional inventory collector(s).

### PR DESCRIPTION
**Pull request by splaspood**
### What does this PR do
* Handle new drives found by additional inventory collector(s)
* Ensure physical drive types are being parsed properly
* Pass entire 'device' object into Drives() collector func

### The HW vendor this change applies to (if applicable)
Any future Drive collectors we add, specifically Marvell storage controllers in this particular case.

### CODE REVIEW

File: actions/inventory.go [hunk 1]
```diff
@@ -187,7 +190,7 @@ func Collect(ctx context.Context, device *common.Device, collectors *Collectors,
 
 // Drives executes drive collectors and merges the data into device.[]*Drive
 // nolint:gocyclo // TODO(joel) if theres more conditionals to be added in here, the method is to be split up.
-func Drives(ctx context.Context, drives []*common.Drive, collectors []DriveCollector) error {
+func Drives(ctx context.Context, device *common.Device, collectors []DriveCollector) error {
```
Comment (joelrebel) [inline actions/inventory.go:193]: This is cleaner than what I had in mind - passing a pointer to the drive slice, I had to check what the code would look like, and it wasn't pretty https://go.dev/play/p/lw5RB8lXdfR

File: utils/mvcli.go [hunk 2]
```diff
@@ -338,6 +338,8 @@ func (m *Mvcli) processDriveType(pdType, ssdType string) string {
 		return common.SlugDriveTypeSATASSD
 	}
 
+	fmt.Println("Got to UNKNOWN")
```
Comment (joelrebel) [inline utils/mvcli.go:341]: purge debug print ? :)

Comment (splaspood) [reply utils/mvcli.go:341]: Debug prints are the seasoning of true champions ;)   Yeah will do.

**Review by joelrebel — APPROVED** [review]
🚀

**Pull request merged**